test/closure.ts

```diff
@@ -64,7 +64,7 @@ function flagsToArgs(flags: Flags): string[] {
 
 /** Run the compiler, asynchronously returning a Result. */
 export function compile(options: Options, flags: Flags): Promise<Result> {
-  const jarPath = options.jarPath || require.resolve('google-closure-compiler/compiler.jar');
+  const jarPath = options.jarPath || require.resolve('google-closure-compiler-java/compiler.jar');
```

Comment:
The jar path is exported officialy https://github.com/google/closure-compiler-npm/blob/834b155375641b80f08464ffee38e58de6c4b764/packages/google-closure-compiler-java/index.js
```suggestion
  const jarPath = options.jarPath || require('google-closure-compiler-java');
```

Reply:
Thanks!  Done.